apps/web/src/app/(default)/dashbord/[slug]/_components/remove-all-registrations-button.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+
+import { useState } from "react";
+import { Dialog, DialogContent } from "@radix-ui/react-dialog";
+
+import { removeAllRegistrations } from "@/actions/remove-all-registrations";
+import { Button } from "@/components/ui/button";
+
+type RemoveAllRegistrationsButtonProps = {
+  slug: string;
+};
+
+export const RemoveAllRegistrationsButton = ({ slug }: RemoveAllRegistrationsButtonProps) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+
+  const closeDialog = () => {
+    setIsOpen(false);
+  };
+  const openDialog = () => {
+    setIsOpen(true);
+  };
+
+  const handleRemoveAllRegistrations = async () => {
+    setLoading(true);
+    setError(null);
+    try {
+      const response = await fetch("/api/remove-registrations", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ slug }),
+      });
```

Comment:
Du kan bare kalle `removeAllRegistrations` direkte her.
```suggestion
      await removeAllRegistrations(id);
```